ports/nrf/boards/electronut_labs_papyr/mpconfigboard.mk

```diff
@@ -0,0 +1,22 @@
+USB_VID = 0x239A
```

Comment:
Do you have your own VID for USB?

Reply:
Nope, I missed that. What do you suggest I put there?

Reply:
Ok, no problem. We'll give you a set of Adafruit PID numbers. So please use `0x003b` for the bootloader, `0x803b` for arduino and `0x803c` for CircuitPython (on the next line.) The VID can remain 0x239A. Thanks for trying CircuitPython!

Reply:
thank you @tannewt! I'll update the PRs.